Ensure null inputs to array setop functions return null output (#19683)
apache/datafusion: docs/source/user-guide/sql/scalar_functions.md

### `array_union`

> Returns an array of elements that are present in both arrays (all elements from both arrays) without duplicates.

Reviewer: ```suggestion
Returns an array containing all unique elements from both input arrays, without duplicates.
```
